server/routes/root.js

```diff
@@ -2,4 +2,6 @@ export default async function (fastify, _opts) {
   fastify.get('/', async function (_request, _reply) {
     return { root: true };
   });
+
+  fastify.register(import('./api/v1/users/index.js'), { prefix: '/user' });
```

Comment:
I don't think we need this, I'm going to remove it, we should just access the user route through it's explicit path: /api/v1/users